### PR TITLE
Enable Request tracer for SEI Integrations

### PR DESCRIPTION
**Pull request by bhapas**
## What does this PR do?

This PR enables request tracer on all SEI Integrations with `httpjson` input. 

Sets the minimum Kibana version to `8.7.1`

Enables the request tracer in system tests.

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Related

- https://github.com/elastic/integrations/pull/5641

### CODE REVIEW

File: packages/akamai/changelog.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "2.7.0"
```
Comment (marc-gr) [inline packages/akamai/changelog.yml:2]: for this one, do we want to get rid of the `-beta`?

File: packages/carbon_black_cloud/manifest.yml [hunk 2]
```diff
@@ -35,6 +35,13 @@ policy_templates:
             title: Hostname
             description: Carbon Black Cloud console Hostname. Find hostname in the console dashboard at the beginning of the web address (Add https:// before the hostname).
             required: true
+          - name: enable_request_tracer
+            type: bool
+            title: Enable request tracing
+            multi: false
+            required: false
+            show_user: false
+            description: The request tracer logs requests and responses to the agent's local file-system for debugging configurations. Enabling this request tracing compromises security and should only be used for debugging.[documentation](https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-input-httpjson.html#_request_tracer_filename) for details.
```
Comment (marc-gr) [inline packages/carbon_black_cloud/manifest.yml:44]: Nit:

```suggestion
            description: The request tracer logs requests and responses to the agent's local file-system for debugging configurations. Enabling this request tracing compromises security and should only be used for debugging. Check the [documentation](https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-input-httpjson.html#_request_tracer_filename) for details.
```
I think it applies for all of them

File: packages/httpjson/data_stream/generic/manifest.yml [hunk 3]
```diff
@@ -14,6 +14,13 @@ streams:
         default: httpjson.generic
         required: true
         show_user: true
+      - name: enable_request_tracer
```
Comment (marc-gr) [inline packages/httpjson/data_stream/generic/manifest.yml:17]: Is it duplicated in this one?

File: packages/ti_misp/data_stream/threat/manifest.yml [hunk 4]
```diff
@@ -11,6 +11,13 @@ streams:
         show_user: true
         default: https://mispserver.com
         description: The URL or hostname of the MISP instance.
+      - name: enable_request_tracer
```
Comment (marc-gr) [inline packages/ti_misp/data_stream/threat/manifest.yml:14]: Also seems duplicated here

File: packages/httpjson/data_stream/generic/manifest.yml (hunk 3 above)
Comment (bhapas) [reply packages/httpjson/data_stream/generic/manifest.yml:17]: possible.. It seems to be added while I was prepping this PR

File: packages/ti_misp/data_stream/threat/manifest.yml (hunk 4 above)
Comment (bhapas) [reply packages/ti_misp/data_stream/threat/manifest.yml:14]: possible.. It seems to be added while I was prepping this PR

File: packages/akamai/changelog.yml (hunk 1 above)
Comment (bhapas) [reply packages/akamai/changelog.yml:2]: @marc-gr  Not sure.. Only `2.6.1-beta` and `2.6.2-beta` exist.. It was sharp until `2.6.0`.. WDYT?

Comment (andrewkroh) [reply packages/akamai/changelog.yml:2]: There are some untested fixes/changes in the package that we are awaiting feedback on before releasing a non-beta. This is where the lack of branching in this project is painful. Could you retain the `-beta`? Then as soon as we get that feedback I will release all these changes to a non-beta version.

File: packages/httpjson/data_stream/generic/manifest.yml (hunk 3 above)
Comment (andrewkroh) [reply packages/httpjson/data_stream/generic/manifest.yml:17]: I found duplicate variable definition at 3 places.

- httpjson/generic/httpjson
- ti_misp/threat/httpjson
- ti_misp/threat_attributes/httpjson

Comment (bhapas) [reply packages/httpjson/data_stream/generic/manifest.yml:17]: Fixed.

But curious if we can make `elastic-package check` catch duplicate variable definitions in manifest files?
@andrewkroh @marc-gr

File: packages/akamai/changelog.yml (hunk 1 above)
Comment (bhapas) [reply packages/akamai/changelog.yml:2]: Made it `-beta`

File: packages/carbon_black_cloud/manifest.yml (hunk 2 above)
Comment (bhapas) [reply packages/carbon_black_cloud/manifest.yml]: Made all the docs consistent.

File: packages/atlassian_bitbucket/data_stream/audit/manifest.yml [hunk 5]
```diff
@@ -52,6 +52,13 @@ streams:
         multi: false
         required: true
         show_user: true
+      - name: enable_request_tracer
+        type: bool
+        title: Enable request tracing
+        multi: false
+        required: false
+        show_user: false
+        description: The request tracer logs requests and responses to the agent's local file-system for debugging configurations. Enabling this request tracing compromises security and should only be used for debugging. See[documentation](https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-input-httpjson.html#_request_tracer_filename) for details.
```
Comment (marc-gr) [inline packages/atlassian_bitbucket/data_stream/audit/manifest.yml:61]: I think a space is missing

```suggestion
        description: The request tracer logs requests and responses to the agent's local file-system for debugging configurations. Enabling this request tracing compromises security and should only be used for debugging. See [documentation](https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-input-httpjson.html#_request_tracer_filename) for details.
```

Comment (bhapas) [reply packages/atlassian_bitbucket/data_stream/audit/manifest.yml]: Fixed

File: packages/httpjson/data_stream/generic/manifest.yml (hunk 3 above)
Comment (andrewkroh) [reply packages/httpjson/data_stream/generic/manifest.yml]: I requested this a while back in https://github.com/elastic/package-spec/issues/421.

Comment (andrewkroh) [reply packages/httpjson/data_stream/generic/manifest.yml]: I checked ae47f49 and did not find any duplicate definitions.

File: packages/akamai/manifest.yml [hunk 6]
```diff
@@ -1,12 +1,13 @@
 name: akamai
 title: Akamai
-version: "2.7.0"
+version: "2.8.0-beta"
+release: ga
```
Comment (andrewkroh) [inline packages/akamai/manifest.yml:4]: You'll need to remove these `release` tags as they are no longer permitted under the new `format_version`.

File: packages/akamai/manifest.yml [hunk 7]
```diff
@@ -1,12 +1,13 @@
 name: akamai
 title: Akamai
-version: "2.7.0"
+version: "2.8.0-beta"
```
Comment (andrewkroh) [inline packages/akamai/manifest.yml:3]: Given that this was already released as 2.7.0, let's remove the `-beta` flag.

File: packages/akamai/manifest.yml (hunk 6 above)
Comment (bhapas) [reply packages/akamai/manifest.yml]: Done

File: packages/akamai/manifest.yml (hunk 7 above)
Comment (bhapas) [reply packages/akamai/manifest.yml]: Done

**Review by marc-gr — APPROVED** [review]

**Pull request merged**